### PR TITLE
SSCS-8894: Carer's Allowance - Airlookup, DWP addresses & Robot Configuration

### PR DESCRIPTION
**Pull request by rasikasri**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-8894
https://tools.hmcts.net/jira/browse/SSCS-9037

### Change description ###

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
     private static final String NO = "No";
     private static final String ESA_CASE_CODE = "051DD";
     private static final String PIP_CASE_CODE = "002DD";
+    private static final String CARERS_ALLOWANCE_CASE_CODE = "070DD";
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java:31]: Don't need this

Comment (rasikasri) [reply src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java]: Removed

**Review by jackmaloney — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java [hunk 2]
```diff
@@ -562,22 +562,22 @@ public SscsPipCaseData getSscsPipCaseData() {
...
 
     @JsonIgnore
-    public Optional<BenefitTypeEnum> getBenefitType() {
+    public Optional<Benefit> getBenefitType() {
         if (appeal != null && appeal.getBenefitType() != null && appeal.getBenefitType().getCode() != null) {
             try {
-                return Optional.of(BenefitTypeEnum.valueOf(appeal.getBenefitType().getCode().toUpperCase()));
-            } catch (IllegalArgumentException e) {
+                return Optional.of(Benefit.findBenefitByShortName(appeal.getBenefitType().getCode().toUpperCase()));
+            } catch (NullPointerException e) {
```
Comment (srochani) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java:569]: Why we are checking null pointer exception here? Ideally we should check if null then do this or else
Here we are suppressing the any other null pointer exception, which I think its not good idea

Comment (rasikasri) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java]: Done

**Pull request merged**